src/screens/Order/OrderUIUtils.res

```diff
@@ -266,24 +269,27 @@ let itemToObjMapper = dict => {
     payment_method_type: getAllPaymentMethodType(dict),
     connector_label: [],
     card_network: dict->getArrayFromDict("card_network", [])->getStrArrayFromJsonArray,
+    customer_id: [],
   }
 }
 
 let initialFilters = (json, filtervalues) => {
   open LogicUtils
 
   let connectorFilter = filtervalues->getArrayFromDict("connector", [])->getStrArrayFromJsonArray
-
+  let {removeKeys} = React.useContext(FilterContext.filterContext)
```

Comment:
we cannot call hooks in a util function, can you please find another way to do this